addons/binding/org.openhab.binding.kodi/ESH-INF/thing/thing-types.xml

```diff
@@ -168,4 +175,16 @@
 		<description>Media type of the current file</description>
 		<state readOnly="true" pattern="%s"></state>
 	</channel-type>
+	<channel-type id="thumbnail" advanced="true">
+		<item-type>String</item-type>
```

Comment:
Very nice to have a way to get Image items from Kodi!
The main reason why they weren't supported in HABPanel is because I never encountered one -
and wasn't sure how to easily get one with real data for testing purposes without Onkyo or Sonos hardware, now it'll be straightforward :wink:

I'll have a look and report back in https://github.com/openhab/org.openhab.ui.habpanel/issues/182, thanks for opening it.